### ISSUE TITLE
Mod doesn't show when starting a new game

### ISSUE THREAD

**Issue by CREEATION** [napalm00/FS19PlaceAnywhere#1]
It shows in the game's "mods" list from the main menu, but not when starting a new save.
What am I doing wrong? :-(

//e: I had to remove "-master" from the filename.

![fs19error](https://user-images.githubusercontent.com/3277769/50172347-25724680-02f5-11e9-8ff2-d8d02c0aa706.jpg)

**Comment by Genimaster** [comment]
If you know how git works, you would have known a cloned download would give you that name.
Since you found this git page, i think you know where to actually find the with mod download also, since that is the first page you come to.

**Comment by CREEATION** [comment]
Not exactly sure why you comment and try to be an ass after I solved the issue myself, but I'll answer anyways.

I found the mod on another site with "-master" appended to the filename, as stated in my edits.

Knowing how git works doesn't relate to not knowing that the game mods aren't allowed to have certain characters and symbols in their filenames and only show in the main mod menu regardless if there is an error in the name.
